ros_buildfarm/config/plot_config.py

```diff
@@ -43,6 +43,15 @@ def __init__(self, name, data):
 
         self.y_axis_exclude_zero = bool(data.get('y_axis_exclude_zero', False))
 
+        self.y_axis_minimum_exists = 'y_axis_minimum' in data.keys()
+        self.y_axis_maximum_exists = 'y_axis_maximum' in data.keys()
```

Comment:
There is no need for a second set of variables. The state that no minimum/maximum exists can be expressed by setting the limits to `None`.

Reply:
Removed bool variables